hack/test-e2e.sh

```diff
@@ -14,8 +17,10 @@ pushd manifests;
      cat ${f} >> ${manifest};
   done;
 popd
+# update the manifest with the image built by ci
+sed -i "s,quay.io/openshift/origin-elasticsearch-operator:latest,${IMAGE_ELASTICSEARCH_OPERATOR}," ${manifest}
 
-sudo sysctl -w vm.max_map_count=262144
+sudo sysctl -w vm.max_map_count=262144 ||:
```

Comment:
the reason we are okay with letting this fail is because for CI something else will set this for us?
Otherwise our ES pods will fail to start up...

Reply:
right - should not have `|| :` here